### PR TITLE
Support Fortran legacy/modern

### PR DESCRIPTION
**Pull request by jake-87**
Couldn't find the template.
See: https://github.com/spenserblack/gengo/issues/34#issuecomment-1695921281

### CODE REVIEW

File: gengo/languages.yaml [hunk 1]
```diff
@@ -219,6 +219,25 @@ Erlang:
     extensions:
       - erl
       - hrl
+"FORTRAN Legacy":
```
Comment (spenserblack) [inline gengo/languages.yaml:222]: Any reason why one is UPPERCASE and the other is Titlecase?

Comment (jake-87) [reply gengo/languages.yaml:229]: Modern fortran is mostly stylised as "Fortran" (as a name) in books/promotional images and whatnot, whereas legacy fortran was almost always stylised as "FORTRAN" (as an abbreviation for "FORmula TRANslation"), so I was following that convention.

Comment (spenserblack) [reply gengo/languages.yaml:229]: Got it, thanks!

**Review by spenserblack — APPROVED** [review]

**Comment by jake-87** [comment]
Apologies for the mistaken alphabet - Wasn't sure what convention you were following.

**Comment by spenserblack** [comment]
No worries, that's why the CI is there. I'm just following how Ruby alphabetizes, but basically it's special characters, then uppercase, then lowercase. If you want to get technical, they're ordered by char code points 🙂

**Pull request merged**